README.md

```diff
@@ -12,7 +12,7 @@ Install locally using `pip install .`
 ## Dependencies:
 
 - [python](https://www.python.org/downloads/)
-- [acxz/congress/python-packaging](https://github.com/acxz/congress/tree/python-package)
+- [unitedstates/congress](https://github.com/unitedstates/congress)
```

Comment:
Suggestion to add a comment like:
`congress` is not yet on pypi, so users must clone and follow `congress` installation instructions [here](https://github.com/unitedstates/congress/blob/main/README.md?plain=1#L46)

Reply:
How about we link to the [Setup](https://github.com/unitedstates/congress#setting-up) section directly?

Don't want to give a line number since upstream could change, but a heading is prob more resilient.

Reply:
```suggestion
- [unitedstates/congress](https://github.com/unitedstates/congress#setting-up)
```

Reply:
Feel free to commit if u agree.